corehq/apps/api/odata/views.py

```diff
@@ -113,7 +113,7 @@ def get(self, request, domain, config_id, **kwargs):
 
         metadata = render_to_string('api/odata_metadata.xml', {
             'fields': case_fields,
-            'primary_key': primary_key,
+            'primary_key': (primary_key,),
```

Comment:
if there are multiple `primary_key`s, shouldn't this field be renamed to `primary_keys`?